### DISCUSSION TITLE
My suggestion is Accounts.

### DISCUSSION THREAD

**Discussion by cheekyrandom** [ScratchAddons/ScratchAddons#5964] [Ideas]
Could you please add an accounts feature where we can sign up and log in? That way, we could have all our add-ons enabled automatically when we use Scratch on different Google accounts. I think this would be very convenient and useful for many Scratchers. Thanks for your hard work and creativity!

**Comment by DNin01** [comment]
Usually, syncing between different accounts isn't supported because typically, one person would only need one account unless they're involved in work or school situations that have integrations or rules that need a separate account to work correctly.

If you want to sync your settings between devices, you can export and import settings files in Scratch Addons' settings. You don't even need download privileges.

**Comment by DNin01** [comment]
If you're asking if settings can sync across devices signed into the _same_ account, then yes, Scratch Addons already features cloud syncing that is handled by the browser, so most browsers should automatically sync your settings as long as you're signed into them.

**Comment by Samq64** [comment]
Can't Chrome already sync settings for all extensions across devices when signed into a Google account?

**Comment by cheekyrandom** [comment]
@DNin01 @Samq64 No idiots, I mean their "own" custom accounts feature

**Reply by Samq64** [reply]
Is your suggestion to have seperate addon profiles when switching between Scratch accounts?

**Comment by cheekyrandom** [comment]
Import and export features, blah blah blah,

<h1>I DONT CARE</h1>

**Comment by cheekyrandom** [comment]
just make the feature dude

**Reply by cally-jt** [reply]
> just make the feature dude

your being really rude and if you dont know js dont just say "just make the feature"

**Reply by WorldLanguages** [reply]
@hamzasolom20 Scratch Addons is open source - feel free to create a fork and make any features you want.

**Reply by Secret-chest** [reply]
@WorldLanguages Please minimise their comments and lock this.

**Reply by WorldLanguages** [reply]
They are blocked from the organization for 1 month

**Comment by CST1229** [comment]
Why do you need an SA-specific accounts feature if browser sync exists?

**Reply by DNin01** [reply]
I think they might have multiple accounts, and modern browsers don't support syncing between different accounts - they only support syncing between devices signed into the same account. That's why this user is suggesting we make our own accounts system - a user could log into the same Scratch Addons account across devices and situations regardless of whether they're using multiple Google accounts.
